vault/core.go

```diff
@@ -783,6 +783,8 @@ func NewCore(conf *CoreConfig) (*Core, error) {
 	c.rawConfig.Store(conf.RawConfig)
 
 	atomic.StoreUint32(c.sealed, 1)
+	metrics.SetGauge([]string{"core", "sealed"}, 1)
```

Comment:
Can we use core.metricSink instead, so that we get a cluster label?

I think that setting the guage only on transitions is not good enough.  Say Vault is up for 60 days but we have a 30 day retention period on metrics.  Then the dashboard stops working.  I think that we need to put this gauge in the emitMetrics goroutine instead.

Reply:
FWIW, Telegraf does remember the value of a gauge for some period of time if it hasn't received an update, but not indefinitely.  That could give us a little extra window, but probably not enough.

Reply:
I see.  I was expecting prometheus-like behavior where the value is polled on each scrape.